ci-operator/step-registry/hypershift/mce/power/heterogeneous/x86-power/hypershift-mce-power-heterogeneous-x86-power-workflow.yaml

```diff
@@ -0,0 +1,37 @@
+workflow:
+  as: hypershift-mce-power-heterogeneous-x86-power
+  documentation: |-
+    The hypershift-mce-power-heterogeneous-x86-power workflow executes tests against a new ephemeral Hypershift heterogeneous agent cluster
+    created on Power and x86 Nodes to validate the cluster creation reaches completed state.
+  steps:
+    pre:
+      - ref: ingress-aws-nlb-manifest
+      - chain: ipi-aws-pre
+      - ref: hypershift-mce-install
+      - ref: hypershift-mce-conf-os-images
+      - ref: hypershift-mce-power-create-hcp
+      - ref: hypershift-mce-power-create-nodepool
+      - chain: hypershift-mce-power-create-ibmcloud-workers
```

Comment:
Using ibmcloud keyword to specify that in case of heterogenous x86 workers are created in VPC service of IBM Cloud.
Have updated the chain description with details.